Source/WebCore/SourcesGTK.txt

```diff
@@ -80,11 +80,14 @@ platform/graphics/gbm/GraphicsContextGLTextureMapperGBM.cpp @no-unify
 platform/graphics/gbm/MemoryMappedGPUBuffer.cpp
 platform/graphics/gbm/PlatformDisplayGBM.cpp @no-unify
 
+platform/graphics/glib/IconGLib.cpp
 platform/graphics/glib/ImageAdapterGLib.cpp
 
 platform/graphics/gtk/ColorGtk.cpp
 platform/graphics/gtk/GdkCairoUtilities.cpp
+platform/graphics/gtk/GdkSkiaUtilities.cpp
 platform/graphics/gtk/IconGtk.cpp
+platform/graphics/gtk/ImageAdapterGtk.cpp
```

Comment:
These are conflicts with [301527@main](https://commits.webkit.org/301527@main) and [301544@main](https://commits.webkit.org/301544@main). You'll need to rebase.

Reply:
Ah, dammit, I thought I had rebased it for this.